Calc: sheet tabs: center tabs

pedropintosilva opened the pull request:
Signed-off-by: Pedro Pinto Silva <pedro.silva@collabora.com>
Change-Id: I467acc1ec81fdccd48d76a52bb7bbc39d28e1ba6

bayramcicek reviewed (approved):
|before|after|
|-|-|
| ![tabs-before](https://github.com/user-attachments/assets/26f2d85c-c75f-4766-8742-c44925cf11a4) | ![tabs-after](https://github.com/user-attachments/assets/01a2348a-df8a-4c71-8d9c-ef01e1791451) |

-compared to previous one- looks good to me on my side. 
it seems(?) tabs are not exactly centered vertically and there is a bit more space on the top(I may be wrong); but this can be ignored(?) I think. (tried to fix this but couldn't find where the issue)

Thanks.

pedropintosilva commented:
> before 	after
> ![tabs-before](https://github.com/user-attachments/assets/26f2d85c-c75f-4766-8742-c44925cf11a4) 	![tabs-after](https://github.com/user-attachments/assets/01a2348a-df8a-4c71-8d9c-ef01e1791451)
> 
> -compared to previous one- looks good to me on my side. it seems(?) tabs are not exactly centered vertically and there is a bit more space on the top(I may be wrong); but this can be ignored(?) I think. (tried to fix this but couldn't find where the issue)
> 
> Thanks.

Thank you for testing it ! I have found another problem. Not really CSS alignment but just the height that we were using, changed 39 to 38 . I have also added a more detailed commit message

pedropintosilva commented:
This is a front end only PR, no need to block this because of cpp analyze test
